sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -561,6 +557,12 @@ static void _send(gnrc_pktsnip_t *pkt, bool prep_hdr)
             return;
         }
 
+        if (gnrc_pkt_len(ipv6) > gnrc_ipv6_netif_get(iface)->mtu) {
+            DEBUG("ipv6: packet too big\n");
+            gnrc_pktbuf_release(pkt);
+            return;
+        }
+
```

Comment:
Oh you meant the check? I did not do anything, I just hit the merge button in my repo :D